### DISCUSSION TITLE
Plugins for VScode maybe

### DISCUSSION THREAD

**Discussion by readytoknow** [squidfunk/mkdocs-material#5329] [Ideas]
Are there any useful plugins to support documentation writing with mkdocs-material? I used markdown extensions in mkdocs to support admonitions and content tabs, but the new grammar for these features broke the original highlighting. So maybe an official plugin for VScode and any other similar text editor will be expected.

**Comment by squidfunk** [comment]
Yeah, that would be amazing!
